depends/Makefile

```diff
@@ -246,6 +247,19 @@ $(host_prefix)/share/config.site : config.site.in $(host_prefix)/.stamp_$(final_
             $< > $@
 	touch $@
 
+$(host_prefix)/share/depends.cmake : depends.cmake.in $(host_prefix)/.stamp_$(final_build_id)
+	@mkdir -p $(@D)
+	sed -e 's|@CONFIG_SITE@|$(host_prefix)/share/config.site|' \
+        -e 's|@CMAKE_SYSTEM_NAME@|$($(host_os)_cmake_system)|' \
```

Comment:
Yes, docs are clear.

FWIW, `CMAKE_SYSTEM_VERSION` is ignored currently: https://github.com/hebasto/bitcoin/blob/455780b1aebae21ebd5d83d2f0ff6a2264ac9608/depends/funcs.mk#L183-L185

Reply:
> Yes, docs are clear.

I think this doc is overgeneralizing. If you look at https://cmake.org/cmake/help/latest/manual/cmake-toolchains.7.html you can see this variable only has meaning when cross compiling for windows and android

Reply:
So it might be Android and Windows, or Android and macOS, or Android and Windows and macOS? In either case it seems like we should figure out if we need to set this.